apps/web/src/pages/integrations/IntegrationsStoreModal.tsx

```diff
@@ -0,0 +1,339 @@
+    setFormIsOpened(false);
+  }
+
+  useEffect(() => {
+    if (integrations) {
+      const initializedProviders = initializeProviders(integrations);
+
+      setEmailProviders(
+        sortProviders(initializedProviders.filter((providerItem) => providerItem.channel === ChannelTypeEnum.EMAIL))
+      );
+      setSmsProvider(
+        sortProviders(initializedProviders.filter((providerItem) => providerItem.channel === ChannelTypeEnum.SMS))
+      );
+
+      setChatProvider(
+        sortProviders(initializedProviders.filter((providerItem) => providerItem.channel === ChannelTypeEnum.CHAT))
+      );
+
+      setPushProvider(
+        sortProviders(initializedProviders.filter((providerItem) => providerItem.channel === ChannelTypeEnum.PUSH))
+      );
+    }
+  }, [integrations]);
+
+  function handleKeyDown(e) {
+    if (e.key === 'Escape') {
+      handleCloseForm();
+    }
+  }
+
+  useEffect(() => {
+    window.addEventListener('keydown', handleKeyDown);
+
+    return () => window.removeEventListener('keydown', handleKeyDown);
+
+    return undefined;
+  }, []);
+
+  return (
+    <Modal
+      withCloseButton={false}
+      closeOnEscape={!isFormOpened}
+      title={
+        <Group style={{ width: '100%' }} position={'apart'}>
+          <Title>Integration Store</Title>
+          <ActionIcon variant={'transparent'} onClick={closeIntegration}>
+            <Close />
+          </ActionIcon>
+        </Group>
+      }
+      classNames={classes}
+      fullScreen
+      opened={openIntegration}
+      onClose={closeIntegration}
+    >
+      <Grid gutter={24}>
+        <Grid.Col lg={isFormOpened ? 8 : 12}>
+          {!isLoading ? (
+            <>
+              <ChannelGroup
+                selectedProvider={selectedProvider}
+                channel={ChannelTypeEnum.EMAIL}
+                providers={emailProviders}
+                title="Email"
+                onProviderClick={handlerVisible}
+              />
+              <ChannelGroup
+                selectedProvider={selectedProvider}
+                channel={ChannelTypeEnum.SMS}
+                providers={smsProvider}
+                title="SMS"
+                onProviderClick={handlerVisible}
+              />
+              <ChannelGroup
+                selectedProvider={selectedProvider}
+                channel={ChannelTypeEnum.CHAT}
+                providers={chatProvider}
+                title="Chat"
+                onProviderClick={handlerVisible}
+              />
+              <ChannelGroup
+                selectedProvider={selectedProvider}
+                channel={ChannelTypeEnum.PUSH}
+                providers={pushProvider}
+                title="Push"
+                onProviderClick={handlerVisible}
+              />
+            </>
+          ) : null}
+        </Grid.Col>
+        <When truthy={isFormOpened}>
+          <Grid.Col lg={4} mt={50}>
+            <IntegrationCardWrapper dark={colorScheme === 'dark'} onKeyPress={handleKeyDown}>
```

Comment:
you don't need to pass the dark prop, because you can destructure the colorScheme from theme in the styled component like this: 
`background-color: ${({ theme }) => (theme.colorScheme === 'dark' ? colors.B20 : colors.BGLight)};`